fields/types/file/FileField.js

```diff
@@ -120,6 +128,20 @@ module.exports = Field.create({
 	// ==============================
 	// RENDERERS
 	// ==============================
+	renderImagePreview () {
+		const { value } = this.props;
+
+		return (
+			<ImageThumbnail
+				component="a"
+				href={this.getImageSource()}
+				target="__blank"
+				style={{ float: 'left', marginRight: '1em' }}
+			>
+				<img src={this.getImageSource()} style={{ height: 90 }} />
```

Comment:
it would be cleaner to do `const src = this.getImageSource()` and then use that in both places